script/DeployExampleTypeCheck.s.sol

```diff
@@ -0,0 +1,61 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity =0.8.19;
+
+import "forge-std/Script.sol";
+import "../src/type-check/ExampleTypeCheck.sol";
+
+contract MyScript is Script {
+    function run() external {
+        uint256 deployerPrivateKey = vm.envUint("PRIVATE_KEY");
+        vm.startBroadcast(deployerPrivateKey);
```

Comment:
let's use cast wallet here as it is safer than having PK on .env
https://twitter.com/RightNowIn/status/1772689521294885321

Reply:
done here [8b41f58](https://github.com/solidity-labs-io/forge-proposal-simulator/pull/47/commits/8b41f581d4023a45613f02546e8a6da69687adf8)